docs/coordinator_guide.md

```diff
@@ -92,7 +92,7 @@ This document will introduce how to deploy Uniffle coordinators.
 |rss.coordinator.dynamicClientConf.enabled|false| whether to enable dynamic client conf, which will be fetched by spark client                                                                                                                                                                                             |
 |rss.coordinator.dynamicClientConf.path|-| The dynamic client conf of this cluster and can be stored in HADOOP FS or local                                                                                                                                                                                          |
 |rss.coordinator.dynamicClientConf.updateIntervalSec|120| The dynamic client conf update interval in seconds                                                                                                                                                                                                                       |
-|rss.coordinator.remote.storage.cluster.conf|-| Remote Storage Cluster related conf with format $clusterId,$key=$value, separated by ';'                                                                                                                                                                                 |
+|rss.coordinator.remote.storage.cluster.conf|-| Remote Storage Cluster related conf with format $clusterId,$key=$value, separated by ';'. If $value contain ',' or '=', you can add '\\' before it. For example: $clusterId,$key=$value1\\,$value2                                                                       |
```

Comment:
https://github.com/apache/incubator-uniffle/blob/0bde6ba30f59060e52a0b857c96538780c05db7b/coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorConf.java#L145-L150

Does it also need to be fixed here?

Reply:
Yes, we need to fixed. I will fix it later.
But we have no conclusion on this proposal. This way is not a graceful proposal. Do we have any better proposal?